<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Jaime Portillo</author>
 <class>Central</class>
 <widget class="QMainWindow" name="Central">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_logo">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>400</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_logo">
       <property name="pixmap">
        <pixmap resource="../../resources.qrc">:/styles/Resources/porti_os.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>70</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="header_background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>70</height>
      </size>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/styles/Resources/header_red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_clock">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>0</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light Condensed</family>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255,0,0);</string>
     </property>
     <property name="text">
      <string>23:23</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bluetoothStatusButton">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>10</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resources.qrc">
       <normaloff>:/bt_icons/Resources/Icons/bt_states/bluetooth_gray.png</normaloff>:/bt_icons/Resources/Icons/bt_states/bluetooth_gray.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_footer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>800</width>
      <height>430</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background:none;
	border:none;
	
}
QPushButton:hover{

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="footer_i">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>260</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="footerButton_1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift Light Condensed</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-angle-double-small-left.png</normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-angle-double-small-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="footerButton_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons_red/Resources/Icons/png-red/play-fill.png</normaloff>:/icons_red/Resources/Icons/png-red/play-fill.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="footerButton_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-angle-double-small-right.png</normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-angle-double-small-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="footer_d">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>260</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>260</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="footerButton_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons_red/Resources/Icons/png-red/icons-menu.png</normaloff>:/icons_red/Resources/Icons/png-red/icons-menu.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="footerButton_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons_red/Resources/Icons/png-red/levels.png</normaloff>:/icons_red/Resources/Icons/png-red/levels.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="footerButton_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-power.png</normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-power.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>footerButton_6</zorder>
     <zorder>footerButton_5</zorder>
     <zorder>footerButton_4</zorder>
    </widget>
    <widget class="QLabel" name="footer_background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/styles/Resources/footer_red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>footer_background</zorder>
    <zorder>footer_i</zorder>
    <zorder>footer_d</zorder>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>480</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_dashboard"/>
    <widget class="QWidget" name="page_player"/>
    <widget class="QWidget" name="page_settings"/>
    <widget class="QWidget" name="page_apps"/>
    <widget class="QWidget" name="page_leds"/>
   </widget>
   <widget class="QFrame" name="frame_volume">
    <property name="geometry">
     <rect>
      <x>652</x>
      <y>200</y>
      <width>40</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame_volume{
	
	background:rgba(57, 57, 57, 200);
	border-radius:20px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="slider_volume">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>30</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QSlider::groove:vertical{
	width:20px;
	background:transparent;
}

QSlider::handle:vertical{
	border:1px solid red;
	background:red;
	height:6px;
	width:20px;
	margin-left:-3px;
	margin-right:-3px;
	border-radius:3px;	

}</string>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>25</y>
       <width>1</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid gray;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>line</zorder>
    <zorder>slider_volume</zorder>
   </widget>
   <widget class="QFrame" name="frame_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame_background{
	background:rgb(20,20,20);
}</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_power">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(24, 24, 24, 200);</string>
    </property>
    <widget class="QFrame" name="frame_powerMenu">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(61, 61, 61);
	border-radius:5px;

}
QPushButton{
	border:2px solid red;
	background-color: rgb(159, 159, 159);	
	border-radius:5px;
}
QPushButton:pressed{
	background:red;
	color:white;
}</string>
     </property>
     <widget class="QPushButton" name="centralRebootButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>280</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>REBOOT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="centralShutdownButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>280</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light Condensed</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SHUTDOWN</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>60</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift Light Condensed</family>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>Power</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>280</width>
         <height>2</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background:red;
border:none;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="powerCloseButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-cross-small.png</normaloff>:/icons_red/Resources/Icons/png-red/fi-rr-cross-small.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>frame_volume</zorder>
   <zorder>frame_power</zorder>
   <zorder>frame_background</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>frame_logo</zorder>
   <zorder>frame_header</zorder>
   <zorder>frame_footer</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
